src/org/omegat/gui/main/MainWindowUI.java

```diff
@@ -279,21 +279,19 @@ public static void initializeScreenLayout(MainWindow mainWindow) {
     }
 
     /**
-     * Assume screen size is 800x600 if width less than 900, and 1024x768 if
-     * larger. Assume task bar at bottom of screen. If screen size saved,
-     * recover that and use instead (18may04).
+     * (20nov22) it looks like we don't have to worry too much about default
+     * size because if the screen is too small for the setting, OmegaT
+     * will just use the whole window size. So I guess we just have to
+     * consider a maximum useable width for big screens.
+     * Let's assume the "standard" is 1920×1080 for "pro" laptops
```

Comment:
```suggestion
     * Let's assume the "standard" is 1920x1080 for "pro" laptops
```
Please don't use a non-Ascii character.